plenigo_plugin/wooCommerce/WC_Gateway_Plenigo.php

```diff
@@ -34,6 +33,8 @@ class WC_Gateway_Plenigo extends \WC_Payment_Gateway {
      * Holds the values to be used in the fields callbacks
      */
     private $options = null;
+    // Available Product Types
+    private $prodTypeList = array('EBOOK', 'DIGITALNEWSPAPER', 'DOWNLOAD', 'VIDEO', 'MUSIC');
```

Comment:
The thing is to iport a classwith unrelated functionality just because we havea const/field that we could reuse...